.github/workflows/tests.yml

```
@@ -0,0 +1,41 @@
+---
+name: Test collection
+
+concurrency:
+  group: ${{ github.head_ref }}
+  cancel-in-progress: true
+
+on:  # yamllint disable-line rule:truthy
+  pull_request:
+    branches: [main]
+  workflow_dispatch:
+
+jobs:
+  ansible-lint:
+    uses: ansible-network/github_actions/.github/workflows/ansible-lint.yml@main
```

```suggestion
    uses: ansible-network/github_actions/.github/workflows/ansible-lint.yml@main
```

Can we move these from ansible-network out into ansible-community somewhere please?  There's a lot of random stuff living under ansible-network and it's really confusing if you're not familiar with the history.

Having them live in ansible-network gives the impression that either they're specifically intended only for the network modules.  Or that the ansible-network group are the sole owners of these actions.  IMHO they would be better "owned" by the community, and people outside of the network folks may be more inclined to contribute to them.

---

cc @Qalthos .

---

I agree with what tremble suggested. @jillr @gravesm?